ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/katex.js

```diff
@@ -17,8 +17,28 @@ katex = mhchemModule(katex);
 function katexStyle() {
 	return [
 		{ name: 'katex.css' },
-		// Note: Katex also requires a number of fonts but they don't need to be specified here
-		// since they will be loaded as needed from the CSS.
+		{ name: 'fonts/KaTeX_AMS-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Caligraphic-Bold.woff2' },
+		{ name: 'fonts/KaTeX_Caligraphic-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Fraktur-Bold.woff2' },
+		{ name: 'fonts/KaTeX_Fraktur-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Main-Bold.woff2' },
+		{ name: 'fonts/KaTeX_Main-BoldItalic.woff2' },
+		{ name: 'fonts/KaTeX_Main-Italic.woff2' },
+		{ name: 'fonts/KaTeX_Main-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Math-BoldItalic.woff2' },
+		{ name: 'fonts/KaTeX_Math-Italic.woff2' },
+		{ name: 'fonts/KaTeX_SansSerif-Bold.woff2' },
+		{ name: 'fonts/KaTeX_SansSerif-Italic.woff2' },
+		{ name: 'fonts/KaTeX_SansSerif-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Script-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Size1-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Size2-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Size3-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Size4-Regular.woff2' },
+		{ name: 'fonts/KaTeX_Typewriter-Regular.woff2' },
+		// Note: Katex also requires a number of fonts and they need to be specified here
+		// since they will be required while exporting to HTML or PDF.
```

Comment:
https://github.com/laurent22/joplin/issues/3078#issuecomment-616158572

@laurent22  As you asked me to add comments I've modified your comment 😄

Reply:
Can you please move the comment after `		{ name: 'katex.css' },` (although in its own line) and change it to:

```
// Note: Katex requires the following fonts to be explicitly specified for exporting
// a note to HTML and PDF.
```

Reply:
Let's mention as well that it's not needed for in-app rendering.

> Declaring the fonts is not needed for in-app rendering because they will be loaded automatically from katex.css. However they must be specified for exporting notes to HTML and PDF, so that they can be bundled with the other assets.

Reply:
> 
> 
> Let's mention as well that it's not needed for in-app rendering.
> 
> > Declaring the fonts is not needed for in-app rendering because they will be loaded automatically from katex.css. However they must be specified for exporting notes to HTML and PDF, so that they can be bundled with the other assets.

Okay sir I'll update the comment with your statement.